### PR TITLE
Add content to "Use the Dashboard"

### PR DESCRIPTION
**Pull request by audrow**
This adds content covered in the demos.

There are a few things that I couldn't reproduce locally, like saving the annotations and uploading multiple archives. I added a note that these may not be working with the newest release. I'm not sure exactly where things were left.

### CODE REVIEW

File: source/How-To-Guides/Use-The-Dashboard.rst [hunk 1]
```diff
@@ -1,7 +1,96 @@
...
 
-TODO:
+Static analysis is a critical step in the formal verification required for safety-critical systems.
+The Space ROS Dashboard is a tool for aggregating and tracking issues discovered through static analysis.
+These issues are represented in `SARIF format <https://sarifweb.azurewebsites.net/>`_, which is a format for representing issues found by static analysis tools.
 
-How to use the Space ROS Dashboard, including installation of the VSCode plug-in.
+The main features of the Space ROS Dashboard are as follows:
 
+- Track the dispositioning issues, including annotating them with links to bug reports in your dispositioning service of choice
```
Comment (Ronoman) [inline source/How-To-Guides/Use-The-Dashboard.rst]: "Track the dispositioning issues" -> "Track issues reported by static analysis tooling"

File: source/How-To-Guides/Use-The-Dashboard.rst [hunk 2]
```diff
@@ -1,7 +1,96 @@
...
+- View a burn-down chart of issues over time
+
+This how-to guide will walk you through the main features of the Space ROS Dashboard.
+
+Prerequisites
+=============
+
+In order to use the Space ROS Dashboard, you will have to install `Visual Studio Code <https://code.visualstudio.com/>`_ and the `Space ROS Dashboard extension <https://marketplace.visualstudio.com/items?itemName=openrobotics.spaceros-dashboard>`_.
+
+You will also need to have SARIF data to view.
```
Comment (Ronoman) [inline source/How-To-Guides/Use-The-Dashboard.rst]: Might want to clarify that this happens through a `colcon build` then a `colcon test`. I've found that I need to either `--cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS` as an extra flag to build, or `set(COLCON_EXPORT_COMPILE_COMMANDS ON)` in the CMakeLists.txt for the respective package to get Cobra to run as well.

File: source/How-To-Guides/Use-The-Dashboard.rst [hunk 3]
```diff
@@ -1,7 +1,96 @@
...
+
+Optionally, you can use `Microsoft's Dev Containers Extension <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers>`_ to use the Space ROS Dashboard with Visual Studio Code inside of a Docker container.
+
+Dispositioning issues
+=====================
+
+Once you have the Space ROS Dashboard extension installed, you can open a SARIF file in Visual Studio Code and the Dashboard will automatically open.
```
Comment (Ronoman) [inline source/How-To-Guides/Use-The-Dashboard.rst]: "open a SARIF file" + " (found in `spaceros_ws/build/<package>/test_results`)"

**Review by Ronoman — CHANGES_REQUESTED** [review]

File: source/How-To-Guides/Use-The-Dashboard.rst (hunk 1 above)
Comment (Ronoman) [reply source/How-To-Guides/Use-The-Dashboard.rst]: Resolved in ef6a1f48

File: source/How-To-Guides/Use-The-Dashboard.rst (hunk 2 above)
Comment (Ronoman) [reply source/How-To-Guides/Use-The-Dashboard.rst]: Addressed in ef6a1f48

**Comment by Ronoman** [comment]
LGTM!

**Review by Ronoman — APPROVED** [review]

**Comment by audrow** [comment]
@Ronoman, I don't have write access on this repo. If you do, would you run the workflow and merge it in?

**Comment by Ronoman** [comment]
@audrow Set it to run. You might want to check in with @nuclearsandwich about repo permissions, but it looks like your blocker here was just being a first-time contributor to the repo (if I read the error message right).

**Pull request merged**